app/controllers/api/v1/orders_controller.rb

```diff
@@ -8,7 +8,7 @@ def index
 
   # POST /api/v1/orders
   def create
-    @order.user = current_user # Automatically associate user
+    @order.user = @current_user # Automatically associate user
```

Comment:
No need to change this, continue to use the `current_user` method.

```suggestion
    @order.user = current_user # Automatically associate user
```